Source/WebCore/platform/graphics/MediaPlayer.cpp

```diff
@@ -2013,6 +2013,17 @@ String MediaPlayer::lastErrorMessage() const
     return m_lastErrorMessage;
 }
 
+String SeekTarget::toString() const
+{
+    StringBuilder builder;
+    builder.append("[");
+    builder.append(WTF::LogArgument<MediaTime>::toString(time));
+    builder.append(WTF::LogArgument<MediaTime>::toString(negativeThreshold));
+    builder.append(WTF::LogArgument<MediaTime>::toString(positiveThreshold));
```

Comment:
Shouldn't you also append some separators between these values?

Reply:
it shows like this `[{"value":2701.603090079002}{"value":2}{"value":5}]` which I thought was good enough as MediaTime already adds `{ }`

Reply:
Alright then, good enough for logging/debugging by people in the know, I suppose. 👍